<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>create_account</class>
 <widget class="QDialog" name="create_account">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>429</width>
    <height>365</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>429</width>
    <height>365</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Создание счета</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>351</width>
     <height>221</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Введите название счета:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="account_name"/>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Выберите валюту счета:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="account_currency">
      <property name="editable">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Введите начальный баланс</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="start_balance"/>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Выберите тарифный план</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="tariff_plan"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>351</width>
     <height>31</height>
    </size>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Введите данные для создания счета&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>300</y>
     <width>351</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="back_button">
      <property name="text">
       <string>Назад</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="create_account_button">
      <property name="text">
       <string>Подтвердить создание счета</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
